src/contractTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/SampleAppConsumerTest.java

```diff
@@ -0,0 +1,164 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.base.Charsets;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Resources;
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.http.MediaType;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.http.HttpStatus.UNPROCESSABLE_ENTITY;
+
+@ExtendWith(PactConsumerTestExt.class)
+public class SampleAppConsumerTest {
+
+    private static final String TEST_S2S_TOKEN = "pact-test-s2s-token";
+
+    @Pact(provider = "sample_app_ocr_validation", consumer = "bulk_scan_orchestrator")
```

Comment:
may be we need to change name of the provider.

Reply:
dang. missed that ;). ta

Reply:
we can make it sth like sample_app and use same provider for all as sample app will not use any state, just we need to mock s2s and call the real system?